src/Components/Profile.js

```diff
@@ -25,8 +25,7 @@ function Profile({ profile, username }) {
             className="p-mr-2 p-mt-2"
           />
         </Avatar>
-        <h1 className="p-m-2">{name}</h1>
-        <h4 className="">({username})</h4>
+        <h1 className="p-m-2">{name} <span class="p-m-2">({username})</span></h1>
```

Comment:
```suggestion
        <div className="p-m-2 p-d-flex">
          <h1 className="p-m-0 p-r-3">{name}</h1>
          <h2 className="p-avatar-username">({username})</h2>
        </div>
```

Reply:
This also fixes the build, not sure about the tests though 😅

Reply:
moreover, this would also fix the failing test case(`react/no-unknown-property`)

Reply:
This unfortunately makes the username lower than the name
![image](https://user-images.githubusercontent.com/17313933/142858850-7eaf5bb8-ea0a-45d7-a856-c1c4cdf30a4d.png)